doc/source/examples/index.rst

```diff
@@ -17,23 +28,19 @@ Here are a series of examples using MAPDL with the ``ansys-mapdl-core`` library.
    :start-line: 2
 
 
-.. === PYMAPDL EXAMPLES ===
+.. === Dev portal examples ==
 
-.. toctree::
-    :hidden:
-    :maxdepth: 3
+.. include:: devportal.rst
+   :start-line: 2
 
-    Verification Manual <https://examples.mapdl.docs.pyansys.com/verif-manual/index.html>
-    Technology Showcases <https://examples.mapdl.docs.pyansys.com/technology_showcase_examples/index.html>
-    extended_examples/index.rst
 
+.. === PYMAPDL EXAMPLES ===
```

Comment:
```suggestion
.. === PyMAPDL examples ===
```
To format this section consistently with Developer Portal examples? (I'm assuming we are using Developer Portal as a proper noun.)